charts/vector/templates/psp.yaml

```diff
@@ -0,0 +1,48 @@
+{{- if and .Values.psp.create (.Capabilities.APIVersions.Has "policy/v1beta1") }}
```

Comment:
This was copied almost entirely from the `vector-agent` chart - aside from some value renaming